Makefile

```diff
@@ -97,11 +112,45 @@ cleanup-ack-req: verify-cluster-variables
 	yq e '.cluster.region=env(CLUSTER_REGION)' -i tests/e2e/utils/ack_sm_controller_bootstrap/config.yaml
 	cd tests/e2e && PYTHONPATH=.. python3.8 utils/ack_sm_controller_bootstrap/cleanup_sm_controller_req.py
 
+test: 
+	$(eval DEPLOYMENT:=vanilla)
+	$(eval DEPLOYMENT_TOOL:=kustomize)
+	@if [ $(DEPLOYMENT) == "vanilla" ]; then \
+	  echo  $(DEPLOYMENT) $(DEPLOYMENT_TOOL); \
+	fi
+
 deploy-kubeflow: bootstrap-ack
-	$(eval DEPLOYMENT_OPTION:=vanilla)
-	$(eval INSTALLATION_OPTION:=kustomize)
-	$(eval AWS_TELEMETRY_OPTION:=enable)
-	cd tests/e2e && PYTHONPATH=.. python3.8 utils/kubeflow_installation.py --deployment_option $(DEPLOYMENT_OPTION) --installation_option $(INSTALLATION_OPTION) --aws_telemetry_option $(AWS_TELEMETRY_OPTION)
+	$(eval DEPLOYMENT:=vanilla)
+	$(eval DEPLOYMENT_TOOL:=kustomize)
+	if [ $(DEPLOYMENT) == "vanilla" ];then \
+	  cd tests/e2e && PYTHONPATH=.. python3.8 utils/kubeflow_installation.py --deployment_option $(DEPLOYMENT) --installation_option $(DEPLOYMENT_TOOL) \
+	fi
+	if [ $(DEPLOYMENT) == "rds-s3" ];then \
+	  @${__VERIFY_ENV__} __VERIFY_ENV__ \
+                CLUSTER_NAME \
+                CLUSTER_REGION \
+		S3_BUCKET \
+		MINIO_AWS_ACCESS_KEY_ID \
+		MINIO_AWS_SECRET_ACCESS_KEY \
+	  PYTHONPATH=.. python3.8 utils/rds-s3/auto-rds-s3-setup.py \
+	    --cluster $(CLUSTER_NAME) \
+	    --region $(CLUSTER_REGION) \
+	    --bucket $(S3_BUCKET) \
+	    --s3_aws_access_key_id $(MINIO_AWS_ACCESS_KEY_ID) \
+	    --s3_aws_secret_access_key $(MINIO_AWS_SECRET_ACCESS_KEY) \
+          cd tests/e2e && PYTHONPATH=.. python3.8 utils/kubeflow_installation.py --deployment_option $(DEPLOYMENT) --installation_option $(DEPLOYMENT_TOOL) \
+        fi
+	if [ $(DEPLOYMENT) == "cognito" ];then \
+	  PYTHONPATH=.. python3.8 utils/cognito_bootstrap/cognito_pre_deployment.py \
```

Comment:
wow! how inconsistent are the ways to install. I dont know if we have proper time before tomorrow to give a unified experience with all these things going on

Reply:
at this point maybe its better to just run the script directly since the user is passing all the necessary flag arguments anyways